goon/code/datums/browserOutput.dm

```diff
@@ -175,7 +175,7 @@ For the main html chat area
 			//Uh oh this fucker has a history of playing on a banned account!!
 			if (found.len > 0)
 				//TODO: add a new evasion ban for the CURRENT client details, using the matched row details
-				message_admins("[key_name(src.owner)] has a cookie from a banned account! (Matched: [found["ckey"]], [found["ip"]], [found["compid"]])")
+				message_admins("<FONT SIZE=+2 span class='danger'>[key_name(src.owner)] has a cookie from a banned account! (Matched: [found["ckey"]], [found["ip"]], [found["compid"]])</span></FONT>")
```

Comment:
get rid of the FONT SIZE=+2 shit and the `</FONT>`
`<FONT>` tags are outdated.  `<span>` tags use CSS, which is more flexible.
Should look like `<span class='danger'>Blah</span>`

Reply:
I already tried span class danger and it wasn't as large as I wanted it to be. See the screenshot above. 
Simply span class danger doesn't increase the size which is why I used font size.

Reply:
There's a `big` class defined in the spreadsheet, you could try something like `<span class='danger big'>`, instead.

Reply:
Going to try that then and we'll see. I just hope it isn't Nar-sie has risen big.